index.js

```diff
@@ -15,7 +15,13 @@ module.exports = {
       }
     ],
     'prefer-promise-reject-errors': 'warn',
-
-    'promise/no-return-wrap': 'off'
+    
+    'promise/no-return-wrap': 'off',
+    
+    // disable rules in prettier scope
+    '@typescript-eslint/object-curly-spacing': 'off',
+    '@typescript-eslint/quotes': 'off',
```

Comment:
https://github.com/radiofrance/eslint-config-radiofrance-typescript/blob/master/index.js

for typescript rule